site2/docs/concepts-messaging.md

```diff
@@ -164,6 +170,12 @@ In the shared and Key_Shared subscription modes, you can negatively acknowledge
 
 Be aware that negative acknowledgment on ordered subscription types, such as Exclusive, Failover and Key_Shared, can cause failed messages to arrive consumers out of the original order.
 
+The following example shows how to call the API :
+
+```
```

Comment:
```suggestion
```java
```

Reply:
Please also apply the change to other code snippets to mark the code block as Java language for correct highlight.

Reply:
It's done